login-module/src/index.js

```diff
@@ -1,90 +1,40 @@
-import constants from './constants';
-import { WebAuth } from 'auth0-js';
-import Quiz from './quiz';
-import Enrollment from './enrollment';
-import Certificate from './certificate';
+import GraphAcademyCore from './core';
 import misc from './misc';
 
-window.GraphAcademyLogin = class GraphAcademyLogin {
+class GraphAcademyPage {
```

Comment:
The main benefit here is that we don't manipulate the access token anymore. We are just calling functions on the "authenticated services" object.